### PR TITLE
Add description for status and metrics

### PR DESCRIPTION
**Pull request by axelsiebert**
Adds documentation for the Spring-Boot-Actuator endpoints and what their purpose will be.

I added a note on the first version of EHRbase supporting this feature which might be release 0.15.0. as an idea how we can also inform the user which version to use at least to get this feature.

### CODE REVIEW

File: source/03_development/08_status_and_metrics/00_security/index.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+.. note:: Added in EHRbase version 0.15.0
+
+Since the Status and Metric endpoints provide such valuable information they are secured by 
+EHRbase integrated security solutions. If you are using at least the *Basic Auth* method to secure
+incoming requests the endpoints are only available for users with the *Admin*-Role.
+
+If you want to access specific metric and health endpoint from a statistics and management service
+ensure not sending the passwords in clear text. For Basic Auth encode them into a Base64 string
+as described here: :doc:`Admin API security<../../07_admin/00_security/index>`.
+
```
Comment (stefanspiska) [inline source/03_development/08_status_and_metrics/00_security/index.rst]: I don't unterstand way this is more secure since you can easy decode base64

Comment (stefanspiska) [reply source/03_development/08_status_and_metrics/00_security/index.rst]: I also don't see any comments on that in  ../../07_admin/00_security/index

**Review by stefanspiska — CHANGES_REQUESTED** [review]
see comments

File: source/03_development/08_status_and_metrics/00_security/index.rst (hunk 1 above)
Comment (axelsiebert) [reply source/03_development/08_status_and_metrics/00_security/index.rst]: I changed the link to the proper documentation in the EHRbase repository.

Also added a note about unsafe usage of Base64 Encoded string and recommend a mor secure Auth Method

**Review by stefanspiska — APPROVED** [review]

**Pull request merged**